scripts/setup/bootstrap.sh

```diff
@@ -32,13 +32,14 @@ _install_additional_pip_requirements() {
         esac
     done
 
-    if ! [ -z "$_SETUP_PLATFORM" ]; then
+    if [ -n "$_SETUP_PLATFORM" ]; then
+        _OLD_IFS=$IFS
+        IFS=","
         if [ -n "$ZSH_VERSION" ]; then
-            IFS="," read -r -A _PLATFORMS <<<"$_SETUP_PLATFORM"
-        else
-            IFS="," read -r -a _PLATFORMS <<<"$_SETUP_PLATFORM"
+            setopt sh_word_split
```

Comment:
Wait, doesn't this need to be `setopt shwordsplit`?

Reply:
well, my change worked with this as-is and did not work without, so maybe they are aliases.

Reply:
Updated anyway as the non-underscore seems to appear more often in docs.